technical/XSD_EXTENSIONS.md

```diff
@@ -0,0 +1,66 @@
+<!--
+SPDX-FileCopyrightText: 2021 Alliander N.V.
+
+SPDX-License-Identifier: CC-BY-4.0
+-->
+
+## XSD Extensions
+For generating the SCL classes, we use the official IEC SCL XSD schemas. These are stored in the [XSD repository](https://github.com/com-pas/compas-scl-xsd). Every week, different Github Actions check for newer version of these XSD schemas. In case newer versions are available, the Github Actions fail and people will get notifications.
+
+In case organizations need some specific information in their SCL files, so called "XSD extensions" can be used. These are separate XSD files containing additional fields or other information that can be used.
```

Comment:
XSD extensions can also be needed by CoMPAS project it-self.

Reply:
True, it's added to the "Project specific XSD extensions" section

Reply:
@Sander3003 I that what you meant?

Reply:
Yes, that works